sygnal/sygnal.py

```diff
@@ -210,9 +210,25 @@ def run(self):
         bind_addresses = self.config["http"]["bind_addresses"]
         pushgateway_api = PushGatewayApiServer(self)
 
-        ensureDeferred(
```

Comment:
This in particular had me scratch my head since...it makes a `Deferred` but never does anything with it.

Reply:
iirc that schedules that coroutine on the Reactor (if I remember the Twisted words correctly).